### PR TITLE
CONFIG: Addition of solution level .editorconfig

### LINKED ISSUE #81: COFIG:  Add an .editorconfig file at Solution Level
This will ensure that code cleanup will place system using directives at the top
This will add the file header template / copyright text to the top of files during code cleanup

These settings are applied via:
```
#Formatting - organize using options

#sort System.* using directives alphabetically, and place them before other usings
dotnet_sort_system_directives_first = true

#Formatting - header template

#add file header template
file_header_template = ---------------------------------------------------------------\nCopyright (c) Coalition of the Good-Hearted Engineers\nFREE TO USE TO CONNECT THE WORLD\n---------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by cjdutoit**
closes #81

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+csharp_new_line_before_open_brace = anonymous_types, methods, control_blocks, types, object_collection_array_initializers
+
+#Formatting - organize using options
+
+#sort System.* using directives alphabetically, and place them before other usings
+dotnet_sort_system_directives_first = true
+
+#Formatting - header template
+
+#add file header template
```
Comment (hassanhabib) [inline .editorconfig:30]: I don't mind the file header. but the other rules should be the default out of Visual Studio.

Comment (cjdutoit) [reply .editorconfig]: @hassanhabib ok, the right click on the solution > add new editor config also exported the other settings.  I have remove everything and only left the two settings on the PR as I have seen them as corrections a few times.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+root = true
+
+# All files
+[*]
+indent_style = space
+
```
Comment (hassanhabib) [inline .editorconfig:7]: I'm not sure that our indentation should be space instead of tab. what's the default?

Comment (cjdutoit) [reply .editorconfig]: @hassanhabib that is the default.   
When you add it with this option
image
it looks like this when you open it in a text editor
image

But I have removed this as as well now so we only have those two settings and none of the defaults.

**Pull request merged**